lib/types/room.js

```diff
@@ -0,0 +1,39 @@
+var Room = function(options) {
+  this.name        = options.name;
+  this.description = options.description;
+  
+  this.players = [];
+
+  this.exit = function(player) {
+    if (!player.room) {
+      return;
+    }
+    var room = player.room;
+    var index = room.players.indexOf(player);
+    room.players.splice(index, 1);
+  };
+
+  this.enter = function(player) {
+    this.exit(player);
+    this.players.push(player);
+    player.room = this;
+  };
+
+  this.fromPerspective = function(player) {
+    return [
+      this.name,
+      '----------',
```

Comment:
this is a fixed length divider but it has above it a variable length string - make the divider 1 'divider character' (-) longer

a small house
---------------
You are standing in front of a small house, sitting atop a green knoll. The doors and windows appear to be board up tightly and the place is unkempt.